extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/DefaultInvoker.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.scheduler.runtime;
+
+import java.util.concurrent.CompletionStage;
+
+import io.quarkus.arc.Arc;
+import io.quarkus.arc.ManagedContext;
+import io.quarkus.scheduler.ScheduledExecution;
+
+public abstract class DefaultInvoker implements ScheduledInvoker {
+
+    @Override
+    public CompletionStage<Void> invoke(ScheduledExecution execution) throws Exception {
```

Comment:
More a question for myself. If I have a method scheduled every second, would I get the same `execution` every time? I'm wondering if there is some kind of "follow up" or if each invocation is completely independent.

Reply:
>  would I get the same execution every time? 

Nope, every execution receives its own `ScheduledExecution` object with different data (e.g. `fireTime`).

Reply:
Thanks!